src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesCrawler.java

```diff
@@ -1,13 +1,17 @@
 package org.jabref.gui.externalfiles;
 
+import java.io.BufferedReader;
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.DirectoryStream.Filter;
 import java.nio.file.Files;
 import java.nio.file.Path;
```

Comment:
I just mentioned you are mixing pre-java7-io api and nio. Is there a special reason?